### PR TITLE
Allow receiving lock operation completion

### PR DESCRIPTION
**Pull request by aryelevin**
Hi there,

I made a small change in order to make sure that the "on" state of a door lock is set only after the lock reports its new state which means the operation completed.

checked with danalock V3.

### CODE REVIEW

**Comment by Smanar** [comment]
Hello, are you sure there is not a problem on your PR ?
When I check "files chnaged" I m seing only one change, adding "if (!isDoorLockDevice)" 2 times but no more "synthax".

**Comment by aryelevin** [comment]
Oh, Hahaha.
I can add a comment.
About your question on the logic side, the point is that as it is now, once I send a REST request to lock/unlock the door-lock device, I immediately get a socket message that the "on" state (which is weird, I would expect to get a "locked" state rather than "on" state) has changed, and this is not correct as the locking process does takes some time, and I need to know when it has done and only then update the state.
Long story short, for making sure the current state is precise, the state of the lock needs to be at the time when the lock reports it and not on the time of the request.

This commit just avoid the report of the state right in the time of the request, so once the operation of the locking will be done, it will be reported via the socket as expected.

**Comment by Smanar** [comment]
Nope, my problem was the missing "{" and "}" :)
Not sure for the "normalized" syntax, but I think there are used in all the code.

**Comment by aryelevin** [comment]
@Smanar :) 
I can understand your concern, A syntax is something each and everyone regulate on a different mind based on different languages guidelines.
As I have a long time experience with C/C++ and embedded projects I sometimes tries to compact the code I use.
Effectively, it will work both ways either with parenthesis or without and no impact on performance, so who's cares.
If you still want me to add parenthesis, just let me know, and I'll be very happy to add it.

**Comment by aryelevin** [comment]
OK, added parenthesis.
Thanks

**Comment by Smanar** [comment]
Was just a personnal remark, I do worst on the code on my side ^^

**Comment by aryelevin** [comment]
Hahaha, OK...
Anyway, thanks.

**Comment by aryelevin** [comment]
Closed duo to moving to a another branch
